var/spack/repos/builtin/packages/bufr/package.py

```diff
@@ -49,6 +55,8 @@ class Bufr(CMakePackage):
     depends_on("py-pip", type="build", when="+python")
     depends_on("py-wheel", type="build", when="+python")
 
+    conflicts("%oneapi@:2024.1", msg="Requires oneapi 2024.2 or later")
```

Comment:
2024.1.2 no good?

Reply:
Correct

Reply:
The issue is with the C bindings (I forget if the bug itself is in icx or ifx or both).

Reply:
Ok, good to know. I remember that I needed to update from 2024.1.0 to 2024.1.2 in order to fix a few bugs. I was hoping this to be the same so that I don't have to install 2024.2.0 ...